### PR TITLE
Paper Tray & Folder Recoloring

### PR DESCRIPTION
**Pull request by Apogee-dev**
paperbins

Adds the ability to recolor paper trays by attacking them with a crayon. Useful if you want to distinguish trays for forms and the like from the generic blank paper supply trays, or just give your front desk a little flair. Note: Medbay paper bins and folders don't use a crayon! Sterilize a folder or paper tray with bleach instead.

:cl:
 * rscadd: Paper trays can now be recolored by using a crayon on them. There's one for every color!

### CODE REVIEW

**Comment by DeityLink** [comment]
I was gonna ask if you made in-hand icons as well for those but I realized they don't have any to begin with, so I guess I'll be doing that right now.

**Comment by Kurfursten** [comment]
You should document the medbay one somewhere, like maybe under bleach on the chemistry page or at least in unsecret tips (maybe put all of the coloring info there)

File: code/modules/paperwork/folders.dm [hunk 1]
```diff
@@ -8,36 +8,88 @@
...
+
+/obj/item/weapon/folder/rainbow
+	crayon = "rainbow"
 
 /obj/item/weapon/folder/update_icon()
 	overlays.len = 0
 	if(contents.len)
 		overlays += image(icon = icon, icon_state = "folder_paper")
+
+	switch(crayon)
```
Comment (Kurfursten) [inline code/modules/paperwork/folders.dm]: better way:
`icon_state = "folder_[crayon]"`

rename the icon state of the medbay one to `folder_sterile`
rename the default one to `folder_`
rename the clown one to `folder_rainbow`

Comment (Apogee-dev) [reply code/modules/paperwork/folders.dm]: Done. Made sure Deity's inhand sprites play nice with the folders, too.

**Comment by Apogee-dev** [comment]
> 
> 
> You should document the medbay one somewhere, like maybe under bleach on the chemistry page or at least in unsecret tips (maybe put all of the coloring info there)

I'll put it in unsecret tips.

**Pull request merged**